pydoctor/test/epydoc/test_restructuredtext.py

```diff
@@ -60,23 +60,22 @@ def prettify(html: str) -> str:
 
 # TESTS FOR NOT IMPLEMENTTED FEATURES
 
-@pytest.mark.xfail
 def test_rst_directive_abnomitions() -> None:
     html = rst2html(".. warning:: Hey")
```

Comment:
Would it be easy to parametrize (as in `pytest.mark.parametrize`) this test so that it runs for all admonitions? I don't think it's worth adding separate test cases just to boost the coverage, but adding a parameter to an existing test case could be worth it.